SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -67,25 +67,34 @@ public ProjectInfoAnalysisResult Execute(AnalysisConfig config, IEnumerable<stri
 
```

Comment:
I couldn't find an easy way to test directly this `Execute()` method - all existing tests will hit the private `ExecuteJavaRunner()` method with an already generated sonar-project.properties file.